### ISSUE TITLE
[tesseract-ui] implement level property selector on sidebar

### ISSUE THREAD

**Issue by frabarz** [tesseract-olap/tesseract-ui#15]
Properties are metadata associated to levels, adding them to a query returns more information associated to each data point.
For the UI, this will probably be a submenu in each drilldown, similar to the popover in cuts, as you can't add properties on levels not in the drilldowns. Feedback would be appreciated.

**Comment by frabarz** [comment]
applied on the `next` branch, set for release on v1.0

**Comment by davelandry** [comment]
@frabarz is this one good to close?

**Comment by frabarz** [comment]
yes, there's a bunch of these issues already implemented in the next branch, but I planned to keep them open until that branch is merged to master
